kotsadm/pkg/gitops/gitops.go

```diff
@@ -134,10 +143,8 @@ func GetDownstreamGitOps(appID string, clusterID string) (*GitOpsConfig, error)
 		if splitKey[2] == "repoUri" {
 			if string(val) == repoURI {
 				// this is the provider we want
-				idx, err := strconv.ParseInt(splitKey[1], 10, 64)
-				if err != nil {
-				}
-				provider, publicKey, privateKey, repoURI, err := gitOpsConfigFromSecretData(idx, secret.Data)
+				idx, _ := strconv.ParseInt(splitKey[1], 10, 64)
```

Comment:
@marccampbell this here looks like it should fail on error and not ignore it.

Reply:
I don't disagree with your comment, but why me?  :)

Reply:
@marccampbell probably because you wrote this block of code 🙂 . i'll update.

Reply:
thanks @salahalsaleh.

Reply:
@marccampbell the red block looks like you wrote it.  but I only looked one commit back.